src/pages/SettingsPage.js

```diff
@@ -69,10 +76,15 @@ export default function SettingsPage({
   }
 }
 
+const PageContainer = styled.main`
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
+`;
+
 const HeadingContainer = styled.header`
```

Comment:
as the basic html construction is:  `header main footer`   i think it is semantic better to use the `header`-tag outside the `main-tag`, maybe an `h1-tag` can do the job inside main without that `header` tag

Reply:
I changed it